src/LineChart.ts

```diff
@@ -70,6 +74,12 @@ module n3Charts {
         eventMgr.on('out.directive', scope.onDatumOut);
       });
 
+      eventMgr.on('legend-click.directive', (series) => {
```

Comment:
Hmmmm I kind of fell short on finding names for these events :-)
Maybe we can delay that while this is still an unreleased beta ? When we have more events, we'll figure out a better naming convention ? Alternatively, if you have ideas, fell free to make some changes.